UI/obs-app.cpp

```diff
@@ -395,6 +396,10 @@ static void do_log(int log_level, const char *msg, va_list args, void *param)
 
 	vsnprintf(str, sizeof(str), msg, args);
 
+	string result = redact(str);
```

Comment:
Should we redact here or maybe only in the log file? Seems like you might want your runtime log output to be non-redacted for the purposes of troubleshooting.

Reply:
The flow here is a bit weird. `str` goes to `LogStringChunk` (file output) but not `def_log_handler` (stdout). I intended for `stdout` to stay non-redacted, which I believe it is, but now I'm paying attention to this `if (IsDebuggerPresent()) {` Windows block where it looks like debug output might be redacted if the debugger is attached on Windows. I may need to take another look and maybe redact slightly differently if thats the case.

Reply:
Alright, I pushed an update that moves the redaction to just before the `LogStringChunk` call, so stdout/debug output should be non-redacted.